### DISCUSSION TITLE
I'm using inpaint on 2x upscaled 1024x1024 images and I'm getting terrible results/artifacts

### DISCUSSION THREAD

**Discussion by rlsdmm** [lllyasviel/Fooocus#2592] [Q&A] [answered]
Is that a bug? what should I do?

**Comment by mashb1t** [comment]
This is not unusual as Stable Diffusion is only trained on specific 1024x1024 and similar sum of pixels aspect ratios.
Also see https://github.com/lllyasviel/Fooocus/pull/1617

**Reply by mashb1t** [reply, marked as answer]
A potential solution would be to use upscaling as one of the last steps in your generation process or decrease the weight / choose another inpainting mode for inpaints on upscaled images.

**Reply by rlsdmm** [reply]
I'll try that, thanks.
